clang-tools-extra/clang-tidy/readability/InconsistentDeclarationParameterNameCheck.h

```diff
@@ -26,7 +26,7 @@ class InconsistentDeclarationParameterNameCheck : public ClangTidyCheck {
                                             ClangTidyContext *Context)
       : ClangTidyCheck(Name, Context),
         IgnoreMacros(Options.getLocalOrGlobal("IgnoreMacros", true)),
-        Strict(Options.getLocalOrGlobal("Strict", false)) {}
+        Strict(Options.get("Strict", false)) {}
```

Comment:
This one sounds like it could just be renamed to `StrictMode`, thoughts? (local-or-global)

Reply:
We can look into it in a separate patch. Renaming is just as problematic as removing it: we break it for users who use `Strict`, so we once again need a grace period where we support both options and then switch to the new one. Personally I don't think it's worth the trouble so I'd just leave it like it is.

Reply:
we should rename it but i think we should do it after final decision of global/local options.